docs/openapi/sites-api.yaml

```diff
@@ -125,3 +125,47 @@ sites-for-organization:
         $ref: './responses.yaml#/500'
     security:
       - api_key: [ ]
+bulk-sites-audit-config:
+    patch:
+      tags:
+          - site
+      summary: Enable audits for multiple sites
+      description: |
+        This endpoint is useful for enabling audit types for multiple sites.
+      operationId: bulkToggleAuditsForSites
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                baseURLs:
```

Comment:
Yes, it is coming from https://github.com/adobe/spacecat-api-service/issues/115, because if we want to easily use the API, we know the baseURL of a site, whereas for the id we need to do another request

Reply:
@iuliag can you elaborate? i don't see in #115 where the requirement/fact that we only have baseURLs comes from. the API usually works with ids.

Reply:
@solaris007 , the consumers generally know the site URL and it's a hassle for them to:

- encode the baseURL
- get the site id via API https://opensource.adobe.com/spacecat-api-service/#tag/site/operation/getSiteByBaseUrl and use that one

In all requests we've received about configuring/looking at some data for a site, only the URL is mentioned, not the site id.